tools/cooja/build.gradle

```diff
@@ -0,0 +1,88 @@
+allprojects {
+  apply plugin: 'java'
+
+    repositories {
+      jcenter()
+    }
+
+}
+
+apply plugin: 'findbugs'
+apply plugin: 'checkstyle'
+apply plugin: 'application'
+apply plugin: 'eclipse'
+apply plugin: 'idea'
```

Comment:
They appear in the building log but they don't appear in the results logs. Anyhow, plenty of errors to fix in the main Cooja before even looking at the apps ;-)